doc/nrf/releases/release-notes-latest.rst

```diff
@@ -93,4 +93,11 @@ The following list summarizes the most important changes inherited from upstream
 
 The following list contains |NCS| specific additions:
 
-*
+* The Bluetooth controllers default uses split architecture
+* NCS Partition Manager (PM) for flash partitioning
+* NCS network socket and address extensions:
+  AF_LTE, NPROTO_AT, NPROTO_PDN, NPROTO_DFU, SOCK_MGMT, SO_RCVTIMEO, SO_BINDTODEVICE, SOL_PDN, SOL_DFU, SO_PDN_CONTEXT_ID, SO_PDN_STATE, SOL_DFU, SO_DFU_ERROR, TLS_SESSION_CACHE,
+  SO_SNDTIMEO, MSG_TRUNC
+  SO_SILENCE_ALL, SO_IP_ECHO_REPLY, SO_IPV6_ECHO_REPLY
+* MQTT: Support enabling of TLS caching
+* CC310: Cryptocell access from non-secure code.
```

Comment:
?
Where was that added? What is the actual change - support for CryptoCell? Or support for access from non-secure code?

Reply:
Support for access from non-secure code:
https://github.com/nrfconnect/sdk-zephyr/commit/32c5d847e8d0196e9b9a32413c67cc98089b1ef2

Reply:
```suggestion
* Updated the nrf9160ns DTS to support accessing the CryptoCell CC310 hardware from non-secure code.
```
Does that make any sense?